src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -25,6 +25,9 @@ internal sealed class BuildServerConnection
 
         // Spend up to 20s connecting to a new process, to allow time for it to start.
         internal const int TimeOutMsNewProcess = 20000;
+        
+        // To share a mutex between processes the name should have the Global prefix
+        private const string GlobalMutexPrefix = "Global\\";
```

Comment:
There is a formatting issue here that is causing the change to fail validation. Not obvious at a glance but formatting locally should fix this. Once that is addressed think the PR is good to go.

Reply:
Fixed. All tests a green. 
Thank you